### PR TITLE
vulnerabilityInfoTab card mode

### LINKED ISSUE #2287: Create cards instead of table for vulnerability page
Right now we have tables for vulnerabilities, we need to make also cards for it

### PR DESCRIPTION
**Pull request by sanyavertolet**
This PR closes #2287

### What's done:
 * Implemented cards for projects on `vulnerabilityInfoTab`
 * Added confirmation on comment deletion
 * Added confirmation on date deletion

### How it looks:

screenshot

### CODE REVIEW

**Review by kgevorkyan — APPROVED** [review]

**Review by Cheshiriks — APPROVED** [review]

**Comment by akuleshov7** [comment]
image

height is too small

**Comment by sanyavertolet** [comment]
> image
> height is too small, need some distance between text and small switchers (code blocks/dots)

How you find this?
screenshot

**Review by akuleshov7 — APPROVED** [review]

File: save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/vulnerability/VulnerabilityController.kt [hunk 1]
```diff
@@ -277,7 +277,7 @@ class VulnerabilityController(
     ): Mono<StringResponse> = vulnerabilityDateDto.toMono()
         // TODO: add permission check
```
Comment (akuleshov7) [inline save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/vulnerability/VulnerabilityController.kt:278]: btw, speaking about permissions, have we limited interactions with vulnerabilities to owner and superadmins?

Comment (sanyavertolet) [reply save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/vulnerability/VulnerabilityController.kt:278]: It still needs to be refactored

**Pull request merged**